Website/Packages/datax-pipeline/src/modules/flowDefinition/flowReducer.js

```diff
@@ -63,6 +70,7 @@ export default (state = INITIAL_FLOW_STATE, action) => {
                 isDirty: true,
                 owner: action.payload,
                 selectedSinkerIndex: 0, // new flow by default contains the metric sinker
+                selectedFlowBatchInputIndex: 0,
```

Comment:
bug: use selectedBatchIndex here instead.

Reply:
selectedFlowBatchInputIndex is correct. this is an index to support multiple inputs. and selectedBatchIndex is an index for the batch job list.